src/tests/unit/tests/DetailsView/reports/components/report-sections/minimal-rule-headertest.tsx

```diff
@@ -2,15 +2,14 @@
 // Licensed under the MIT License.
```

Comment:
The rename of this file looks like it's missing a "." before the "test.tsx"

Reply:
good catch! Updated.